homeassistant/components/unifi/sensor.py

```diff
@@ -96,6 +98,12 @@ def async_device_outlet_power_supported_fn(
     return controller.api.outlets[obj_id].caps == 3
 
 
+@callback
+def async_device_outlet_supported_fn(controller: UniFiController, obj_id: str) -> bool:
```

Comment:
Can we really consider one being representative of the other? i.e. should we have two methods or is this good enough?

Reply:
I beleive one is good enough. I only have data on the PDU-Pro to go by, which i beleive is the only device currently that supports this.

The UI uses both of these fields to display a utilization %, so I felt it was safe to assume if balance was available, consumption would be as well.

Reply:
maybe it makes sense to check of both properties have values in this function?

Reply:
Either we do two methods of expect one value to be enough to determine availability